### PR TITLE
test(text-field): adding additional examples in form page

### PR DESCRIPTION
**Pull request by theJohnnyMe**
- adding phone and address input in different states

Note: we would need to add other native props, like type="number", valid, min etc in our text field component.

### CODE REVIEW

File: src/pages/FormPage.tsx [hunk 1]
```diff
@@ -62,6 +65,39 @@ const FormPage = () => {
...
+                name="homeAddress"
+                label-position="outside"
+                no-min-width
+                placeholder="Majorvägen 32"
+                helper="Unknown address"
+              >
+                <span slot="prefix">
+                  <tds-icon name="pin" size="16px"></tds-icon>
+                </span>
+              </tds-text-field>
```
Comment (mJarsater) [inline src/pages/FormPage.tsx:98]: Should we try to set this one in an error state if the text field is empty? So in handleSubmit we check if it's empty and if so we don't submit the form and we set this one into an error state.

Comment (theJohnnyMe) [reply src/pages/FormPage.tsx:98]: Great idea, will do it ;)

**Comment by theJohnnyMe** [comment]
Also, the improvement to validation is to check it on focus/blue instead, so feedback is instant. But I would say lets do it for all fields as a task to create validation logic for every input element.

**Review by mJarsater — APPROVED** [review]
Great! I was gonna suggest to add checks on focus/blur but saw your comment on doing it in a separate ticket

**Pull request merged**